python/pyspark/__init__.py

```diff
@@ -16,7 +16,7 @@
 #
 
 """
-PySpark is the Python API for Spark.
+PySpark is the Python API for Spark
```

Comment:
Is this intentional change?

Reply:
yup. I will revert this one too. just intended to test Python tests only since Scala tests takes long.